docs/_docs/framework/capabilities/optimize/architecting.md

```diff
@@ -87,6 +87,11 @@ At this point, you should have a clear architectural guidance. As you move beyon
 
 This capability is a part of the FinOps Framework by the FinOps Foundation, a non-profit organization dedicated to advancing cloud cost management and optimization. For more information about FinOps, including useful playbooks, training and certification programs, and more, see the [Architecting for cloud](https://www.finops.org/framework/capabilities/architecting-for-cloud/) article in the FinOps Framework documentation.
 
+You can also find related videos on the FinOps Foundation YouTube channel:
+
+<!--[!VIDEO https://www.youtube.com/embed/zMonuvmZE1g&list=PLUSCToibAswm62kf2eILBPaRHobvxNy35&pp=iAQB]-->

```

Comment:
This link is broken. Is this what you tried to add? https://www.youtube.com/watch?v=E5OJmjrKYXk

Reply:
I just confirmed. The link is correct. Looks like that video simply cannot be embedded 😕